<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor_form</class>
 <widget class="QWidget" name="monitor_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="monitor_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="temperature1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>155</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="temperature2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>155</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>temperature</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hud1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>56.5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="smoke2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>355</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>smoke</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_smo2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="stepping1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stepping1</string>
   </property>
  </widget>
  <widget class="QLabel" name="humidity2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>255</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>humidity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tem1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>26.5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="humidity1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>255</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>humidity</string>
   </property>
  </widget>
  <widget class="QLabel" name="smoke1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>355</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>smoke</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepping2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stepping2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hud2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_smo1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tem2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mark">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>@Zhousheng 2023.05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>refresh.png</normaloff>refresh.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>refresh.png</normaloff>refresh.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
